scripts/userstyles.yml

```diff
@@ -683,6 +683,13 @@ userstyles:
     readme:
       app-link: "https://tabnews.com.br"
       current-maintainers: [*Guaxinim5573]
+  tldraw:
+    name: tldraw
+    categories: [productivity]
+    color: text
+    readme:
+      app-link: "https://www.tldraw.com"
+      current-maintainers: [*genshibe]
```

Comment:
might be worth noting that the exports are not in the catppuccin theme in the FAQ section

Reply:
```suggestion
    readme:
      app-link: "https://www.tldraw.com"
      usage: |+
        > [!NOTE]
        > This theme only themes the colors on the tldraw website. Exported graphics **will not be Catppuccin-themed**.
      current-maintainers: [*genshibe]
```

> [!NOTE]
> This theme only themes the colors on the tldraw website. Exported graphics **will not be Catppuccin-themed**.